docs/openapi.yaml

```diff
@@ -145,6 +180,7 @@ paths:
                     domain: "admin"
                   }
                   ]
+                  programRoles: []
```

Comment:
POST /users response example is missing the active field

Reply:
changed and pushed